src/main/resources/templates/sightings.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html layout:decorate="layout"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <title>Sightings</title>
+    <link th:href="@{/css/sightings.css}" rel="stylesheet" />
+</head>
+<body class="page-container">
+<section class="sightings-section" layout:fragment="content">
+    <div class="sightings-heading">
+        <h1>Sightings</h1>
+    </div>
+    <div class="sightings-results-section">
+        <ul class="sightings-list">
+            <li class="results-list">
+                <div class="sightings-list" th:each="sighting: ${approvedSightings}">
+                    <div class="result-item">
+                        <br>Date: <td th:text="${sighting.date}" /></br>
+                        <br>Latitude: <td th:text="${sighting.latitude}" /></br>
+                        <br>Longitude:  <td th:text="${sighting.longitude}" /></br>
```

Comment:
Separate note - `<br>` are _just_ page breaks, wrapping them around something doesn't really mean that much. You generally see them on their own as `<br>` or self-closing as `<br />`